eng/xplat-job.yml

```diff
@@ -40,9 +40,9 @@ jobs:
       ${{ if and(eq(parameters.osGroup, 'Linux'), ne(variables['System.TeamProject'], 'public')) }}:
         name: dnceng-linux-internal-temp
       ${{ if and(eq(parameters.osGroup, 'OSX'), ne(variables['System.TeamProject'], 'public')) }}:
-        name: Hosted Mac Internal
+        name: Hosted Mac Internal Sierra
       ${{ if and(eq(parameters.osGroup, 'OSX'), eq(variables['System.TeamProject'], 'public')) }}:
-        name: Hosted MacOS
+        name: Hosted Mac Internal Sierra
```

Comment:
Unless I'm mistaken we probably don't want to use a pool named "internal" on the public project.

Reply:
The hosted pools are cleaned after every use.  I am working with the Mac team to handle this better but for now this will unblock.

Reply:
I am testing another solution in https://github.com/dotnet/coreclr/pull/21816 which should allow to use previous name of pools